AutoCollection/Performance.ts

```diff
@@ -200,18 +155,15 @@ class AutoCollectPerformance {
                 this._client.trackMetric(name + "user", user / total);
             }
 
-            var combinedName = "% total cpu ";
+            var combinedName = "% total cpu";
             var combinedTotal = (totalUser + totalSys + totalNice + totalIdle + totalIrq) || 1;
 
-            this._client.trackMetric(combinedName + "user", totalUser / combinedTotal);
-            this._client.trackMetric(combinedName + "sys", totalSys / combinedTotal);
-            this._client.trackMetric(combinedName + "nice", totalNice / combinedTotal);
-            this._client.trackMetric(combinedName + "idle", totalIdle / combinedTotal);
-            this._client.trackMetric(combinedName + "irq", totalIrq/ combinedTotal);
-
-            // todo: remove this legacy counter once the UI updates (~june 2015)
-            this._trackLegacyPerformance(PerfCounterType.ProcessorTime, totalUser / combinedTotal);
-            this._trackLegacyPerformance(PerfCounterType.PercentProcessorTime, (combinedTotal - totalIdle) / combinedTotal);
+            this._client.trackMetric("\\Processor(_Total)\\% Processor Time", (combinedTotal - totalIdle) / combinedTotal);
```

Comment:
Following more offline discussions, common perf counters with .NET SDKs will be reported using the metric names they currently use until a new set is decided. There should be no user impact, as these are converted into appropriate values in the performanceCounter Analytics table by the backend